Official docker image is broken

A recent version of Argus moved the `argus.ws` module to another submodule (not sure which one, atm).  However, the Docker images that are automatically built and published on each release contain this entrypoint script, which causes the image to fail on startup:

https://github.com/Uninett/Argus/blob/f567631ef3acb5a4e003b9057ceccd13d1b77e91/docker/docker-entrypoint.sh#L6


This should be fixed, but there should also be some test in the workflow that fails the build if the resulting image cannot be started.

The above is specifically an error from the argus-docker Docker Compose-based project.  Running with less fluff, this is error I get:

```
[2025-04-04 06:27:11 +0000] [7] [INFO] Starting gunicorn 23.0.0
[2025-04-04 06:27:11 +0000] [7] [INFO] Listening at: http://0.0.0.0:8000 (7)
[2025-04-04 06:27:11 +0000] [7] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2025-04-04 06:27:11 +0000] [38] [INFO] Booting worker with pid: 38
[2025-04-04 08:27:11 +0200] [38] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/gunicorn/arbiter.py", line 608, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.10/site-packages/uvicorn/workers.py", line 75, in init_process
    super().init_process()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/workers/base.py", line 135, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/workers/base.py", line 147, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/base.py", line 66, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 57, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 47, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.10/site-packages/gunicorn/util.py", line 370, in import_app
    mod = importlib.import_module(module)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/local/lib/python3.10/site-packages/argus/spa/ws/asgi.py", line 20, in <module>
    from .consumers import OpenIncidentConsumer
  File "/usr/local/lib/python3.10/site-packages/argus/spa/ws/consumers.py", line 2, in <module>
    from channels.generic.websocket import JsonWebsocketConsumer
ModuleNotFoundError: No module named 'channels'
[2025-04-04 08:27:11 +0200] [38] [INFO] Worker exiting (pid: 38)
[2025-04-04 06:27:11 +0000] [7] [ERROR] Worker (pid:38) exited with code 3
[2025-04-04 06:27:11 +0000] [7] [ERROR] Shutting down: Master
[2025-04-04 06:27:11 +0000] [7] [ERROR] Reason: Worker failed to boot.
[INFO  tini (1)] Spawned child process '/api-entrypoint.sh' with pid '7'
[INFO  tini (1)] Main child exited normally (with status '3')
```

--- a/changelog.d/1310.fixed.md
+++ b/changelog.d/1310.fixed.md
@@ -1,0 +1,1 @@
+Fix broken Docker images to still work with SPA front-end

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -15,7 +15,7 @@ RUN mkdir -p /static && chown argus /static
 ENV STATIC_ROOT=/static
 
 COPY . /src
-RUN pip install /src && rm -rf /src
+RUN pip install '/src[spa]' && rm -rf /src
 
 # Install API backend settings suitable for Docker deployment
 RUN mkdir /extrapython

